index.qmd

```diff
@@ -1,20 +1,36 @@
 ---
-title: "DaVinCI"
+title: "DaVinci"
 ---
 
-## Dynamic Visualization for Clinical Insights
+## Dynamic Visualization for Clinical Insights (DaVinci)
 
-### What is DaVinCI?
+### What is DaVinci?
 
+DaVinci provides tools to review, aggregate and visualize data to develop and deliver safe and effective treatments for patients. Instead of creating entire R/Shiny applications, DaVinci provides individual modules. Modules are functional building blocks that can be linked together into customized apps. This customization enables App Creators to build their own app according to trial-specific needs without deep R knowledge. The concept of developing loosely coupled, functional units and composing them into a larger application is subsequently referred to as modular approach.
 
-DaVinCI offers R/Shiny based tools to review, aggregate and visualize data arising from clinical trials, supporting the goal of developing and delivering safe and effective treatments for patients. However, instead of creating entire R/Shiny applications, DaVinCI provides individual modules. Modules are functional building blocks that can be linked together into customized apps. This customization enables App Creators to build their own app according to trial-specific needs without deep R knowledge. 
+### The modular approach
 
+To create an app you need to identify which modules to include in the first place. Think of each DaVinci module as individual building stone that you can put together to build a vehicle for a certain purpose. If you want to go fast, build a motorcycle, if you want to go far build a common car. The vehicle refers to the DaVinci app that you create according to your use case.
 
-### Our commitment and contribution to open-source
- 
-DaVinCI software development including the codebase  will be made fully open-source, which will allow you to use or co-develop DaVinCI.  
-Our initial open-source release is planned by **Q3-2024!**  
-Don't miss our release and other DaVinCI related news, and connect with our Product Owner [Stefan Doering](https://www.linkedin.com/in/stefan-d%C3%B6ring-6134a595/){target="_blank"} on LinkedIn!
+![](quarto/images/building_blocks.png){fig-align="center"}
 
 
+For that purpose, modules are provided as R packages. One package contains one or multiple modules. Choose the modules freely according to the purpose of your visualization. For data cleaning purposes you might not need a heatmap, but you may want to incorporate it for biomarker analyses. Safety review might benefit from an eDISH plot, and all of them probably want to display listings as well. These are only suggestions, modules can be combined freely. Some of them are even able to communicate with each other. 
```

Comment:
```suggestion
DaVinci modules are provided as R packages, with each package containing one or multiple modules. Choose the modules freely according to the purpose of your visualization. For instance, while a heatmap may not be necessary for data cleaning, it could be useful for biomarker analyses. Safety review might benefit from an eDISH plot, and listings are likely to be needed in all use cases. These are only examples, modules can be combined as needed. Some modules even have the capability to interact with each other. 
```